src/os/bluestore/BlueStore.cc

```diff
@@ -7441,6 +7441,12 @@ int BlueStore::expand_devices(ostream& out)
           << std::endl;
       }
     }
+
+    // we grow the allocation range, must reflect it in the allocation file
+    dout(0) << __func__ << "::NCB:: old-size=" << size0 << ", new-size=" << size << dendl;
+    alloc->init_add_free(size0, size-size0);
+    need_to_destage_allocation_file = true;
+
```

Comment:
Is it possible to get rid of 
```
dout(5) << __func__ << "::NCB::calling umount()" << dendl;
```
while you are modifying this function ?

Reply:
unrelated to this change